buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+repositories {
+    google()
+    maven(url = "https://plugins.gradle.org/m2/")
+    mavenCentral()
+    jcenter()
+}
+
+plugins {
+    java
+    `kotlin-dsl`
+}
+
+dependencies {
+    implementation(gradleApi())
+    implementation("com.android.tools.build:gradle:3.6.3")
```

Comment:
implementation Libraries.gradlePlugin
    implementation Libraries.jacoco
    implementation Libraries.kotlin

Strange... it should be an entry to `Libraries.gradlePlugin` on `dependencies.gradle` :thinking: